subgraph/auxiliary/subgraph.yaml

```diff
@@ -0,0 +1,23 @@
+specVersion: 0.0.2
+schema:
+  file: ./schema.graphql
+dataSources:
+  - kind: ethereum/contract
+    name: MosaicFacilitator
```

Comment:
name should contain start gateway?

Reply:
This needs to be specific contract name. Datasources name are specific contract names.  I have updated the same. 
e.g. https://github.com/graphprotocol/decentraland-subgraph/blob/master/subgraph.yaml